codespell_lib/_codespell.py

```diff
@@ -411,8 +417,20 @@ def ask_for_word_fix(line, wrongword, misspelling, interactivity):
     return misspelling.fix, fix_case(wrongword, misspelling.data)
 
 
+def print_context(lines, index, context_before, context_after):
+    if context_before < 0:
+        return
```

Comment:
How can `context_before` ever be `< 0`? This seems like it should not happen. If so, inputs should be sanity checked / fixed so this conditional is no longer necessary.

Reply:
`context_before` is negative if no context option is used at all. But I see that this may be confusing. I'll try to refactor the code a bit.